examples/vue-example-app/src/router/index.js

```diff
@@ -0,0 +1,80 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Home from '../views/Home.vue'
+
+Vue.use(VueRouter)
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
+  },
+  {
+    path: '/demo',
+    name: 'Demo',
+    component: () => import(/* webpackChunkName: "demo" */ '../views/Demo.vue')
```

Comment:
Is there any reason why Demo and Home are in views directory and not in pages?

Reply:
I think it makes sense to move all these in pages, I will do that. I will remove package-lock.json as well.